LeagueToolkit/IO/SimpleSkinFile/SimpleSkin.cs

```diff
@@ -166,9 +166,9 @@ public void Write(Stream stream)
                     bw.Write((uint)SimpleSkinVertexType.Basic);
                 }
 
-                R3DBox box = GetBoundingBox();
-                box.Write(bw);
-                box.GetBoundingSphere().Write(bw);
+                Vector3[] allVertexPositions = this.Submeshes.SelectMany(submesh => submesh.Vertices).Select(vertex => vertex.Position).Distinct().ToArray();
```

Comment:
As in, a `.Distinct()` call is always made inside the calculate functions with no assumptions made about distinctness of the input? I can do that.
If you mean an actual flag in the function call though, I don't see how it would help in any way to not have distinct input points; the end result shouldn't differ.